.github/workflows/java.yml

```diff
@@ -45,7 +45,7 @@ jobs:
   
   test:
     if: startsWith(github.ref, 'refs/tags/') != true
-    runs-on: ubuntu-18.04
+    runs-on: macos-latest
```

Comment:
why macos?

Reply:
Because some tests were failing for no apparent reason on linux. we have changes it to macox for now. will fix it later as part of a separate ticket